contracts/FreakTeam.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
```

Comment:
Why are there so many empty lines? 
Please follow the stated Solidity Style Guide (you can enforce this with solhint) - https://docs.soliditylang.org/en/v0.8.6/style-guide.html